src/Farmer/Common.fs

```diff
@@ -987,3 +987,7 @@ module Roles =
         let NetworkContributor = RoleID "4d97b98b-1d4f-4787-a291-c67834d212e7"
         let PrivateDnsZoneContributor = RoleID "b12aa53e-6015-4669-85d0-8515ebb3ae7f"
         let TrafficManagerContributor = RoleID "a4b10055-b0c7-44c2-b00f-c7b5b3550cf7"
+
+module Dns =
+    type DnsRecordType = Unknown | A | AAAA | CName | NS | PTR | TXT | MX
+    // | Soa | Srv | CAA
```

Comment:
Can we remove these commented out entries?

Reply:
Done